Makefile

```diff
@@ -13,6 +13,7 @@ LIBEXECDIR ?= ${PREFIX}/libexec
 MANDIR ?= ${PREFIX}/share/man
 ETCDIR ?= ${DESTDIR}/etc
 ETCDIR_CRIO ?= ${ETCDIR}/crio
+SHAREDIR ?= ${PREFIX}/share/containers
```

Comment:
I think the canonical name for `${PREFIX}/share` is [`datarootdir`][1].  Do we want to respect that if the caller sets it?  I'm not sure about the casing; have they always recommended lowercase?  I've certainly seen `PREFIX` more often than `prefix`, and [variable names are case-sensitive][2].  Haven't had time to blame the make source though…

[1]: https://www.gnu.org/software/make/manual/html_node/Directory-Variables.html#index-datarootdir
[2]: https://www.gnu.org/software/make/manual/html_node/Using-Variables.html

Reply:
rpm calls it the datadir.

 %_datadir		%{_prefix}/share
autoconf uses datarootdir

I don't really care...

Reply:
Gnu defines: So I guess I will change, although I will keep it upper case, to be consistent. 
```
‘datarootdir’

    The root of the directory tree for read-only architecture-independent data files. This should normally be /usr/local/share, but write it as $(prefix)/share. (If you are using Autoconf, write it as ‘@datarootdir@’.) ‘datadir’’s default value is based on this variable; so are ‘infodir’, ‘mandir’, and others.
```